bee-message/src/lib.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! Core data types for messages in the tangle.
+
+//#![deny(missing_docs, warnings)]
+
+#![no_std]
+#![warn(missing_docs)]
+
+extern crate alloc;
+
+#[macro_use]
+mod macros;
+mod message;
+mod message_id;
+
+/// A module that provides types and syntactic validations of addresses.
```

Comment:
Ignore if you want:
My personal preference nowadays is to put the module documentation WITHIN the module itself. This allows us to expand the documentation as we please and add code examples etc to it without cluttering this `lib.rs`. The documentation reader will almost always go through `docs.rs` because it's interlinked. So if the intention here is to have everything explained in a single place, then this not so important I think.

Reply:
That's what's been done in all the other public-facing modules here as well, so I'll update to make it more consistent 👍